test/e2e/framework/nodes_util.go

```diff
@@ -74,17 +74,22 @@ func etcdUpgradeGCE(target_storage, target_version string) error {
 }
 
 func ingressUpgradeGCE(isUpgrade bool) error {
-	// Flip glbc image from latest release image to HEAD to simulate an upgrade.
-	// Flip from HEAD to latest release image to simulate a downgrade.
-	// Kubelet should restart glbc automatically.
 	var command string
 	if isUpgrade {
-		// Upgrade
-		command = "sudo sed -i -re 's/(image:)(.*)/\\1 gcr.io\\/k8s-ingress-image-push\\/ingress-gce-e2e-glbc-amd64:latest/' /etc/kubernetes/manifests/glbc.manifest"
+		// User specified image to upgrade to.
+		targetImage := TestContext.IngressUpgradeImage
+		targetImage = strings.Replace(targetImage, "/", "\\/", -1)
```

Comment:
You can do 

```
fmt.Sprintf(`sudo sed -i -re 's|(image:)(.*)|\1 %s|`)
```

and avoid having to escape the `/` char above.

Reply:
Done